fantasy-docs.md

```diff
@@ -362,6 +393,37 @@ If your function accepts 0 parameters, then it can be visualized easily. But how
 
 This is a really powerful way to let consumers customize the goods you've designed before buying or manufacturing them. For example, you might put a design for a cool 3D printed office chair on thingiverse.com which has a function `main(name: Text)`. This function describes a chair with the given name embossed into the back. When a consumer wants to 3D print it, the 3D printing service will let them input their desired name, view the chair with that name embossed, and then order it.
 
+## Tests
+Functions are marked as tests using the `#[test]` attribute. They're run via the KittyCAD CLI or by the test runner built into the KittyCAD modeling app.
+
+```kcl
+#[test]
+division_by_1_doesnt_change_number() =
```

Comment:
I might be mistaken, but I think this isn't the new function syntax you outlined right? I thought it would be:
```kcl
#[test]
division_by_1_doesnt_change_number = () => 
    assert_eq(4/1, 4)
```

or even
```kcl
#[test]
division_by_1_doesnt_change_number = assert_eq(4/1, 4)
```
because of the syntactic sugar?

Reply:
You're right, this was from before I changed the syntax :)